apps/amplify-demo/.gitignore

```diff
@@ -27,6 +27,7 @@ yarn-error.log*
 # amplify
 src/react-app-env.d.ts
 src/graphql
+src/API.ts
 amplify/
```

Comment:
Question: we already has this `amplify/`, why do we need that `#amplify-do-not-edit` section?

Reply:
So when I upgraded to aws-amplify 4.x.x, it got re-added, I have deleted duplicates thanks for the catch.